<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QAction::unchecked {background-color:red} QAction::checked {background-color:green}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="3">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnStop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="DragProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%m%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btnPause">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="styleSheet">
       <string notr="true">QLineEdit QPushButton {backgroun-color:rgb(170, 0, 255)}</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="3">
     <widget class="QLabel" name="labelArtImage">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开音频文件</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>打开音频目录</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>切换英文</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>切换中文</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragProgressBar</class>
   <extends>QProgressBar</extends>
   <header location="global">dragprogressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
